.babelrc

```diff
@@ -1,19 +1,21 @@
 {
   "presets": [
-    "es2015",
-    "react",
-    "stage-3"
+    [
+      "@babel/preset-react"
+    ],
+    [
+      "@babel/preset-env",
```

Comment:
this setting does most of what the older ones were doing (except for `class-properties`)